source/configuration_manual/mail_location/obox/s3.rst

```diff
@@ -4,8 +4,13 @@
 S3-compatible Storages
 ======================
 
-This document covers the S3 compatible storage services. In case you want to
-use AWS S3 please also refer to :ref:`amazon_s3`.
+Supported providers
+-------------------
+
+For Dovecot Pro edition, only AWS S3 is officially supported.
+Please see :ref:`amazon_s3`.
+
+This document covers the S3 compatible storage services, which are not supported.
```

Comment:
I'm not clear on what this means. What purpose serves this doc if the stuff in it isn't actually supported? 

Or is this a documentation explaining how to configure Dovecot for any S3 storage, even untested ones? Does AWS's S3 works with that as well independently of the dedicated (and pro version only) AWS S3 plugin?

I find this confusing and not sure what it all means, just my 2 cents. :)